lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,48 @@
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTOGONALITIES = %i[length_x length_y].freeze
+        CUT_DIRECTION_ERROR = 'Not valid cut direction, should one of thes: :length_x :length_y'.freeze
+
+        def initialize(column:, footing:, effective_height:, load_from_column:, cut_direction:)
+          @column = column
+          @footing = footing
+          @load_from_column = load_from_column
+          @effective_height = effective_height.to_f
+
+          raise ArgumentError, CUT_DIRECTION_ERROR unless ORTOGONALITIES.any?(cut_direction)
+
+          @cut_direction = cut_direction
+        end
+
+        def solicitation_load
+          solicitation * @footing.send(ortogonal_direction)
```

Comment:
let's use `public_send` instead of `send` on this and all the use cases below. The difference is that `public_send` works only with public methods and `send` works with all of them

Reply:
done!